src/qpython_imageprovider.cpp

```diff
@@ -141,12 +145,12 @@ QPythonImageProvider::requestImage(const QString &id, QSize *size, const QSize &
     // While we could re-pack the data to be aligned, we don't want to do that
     // for performance reasons.
     // If we're using 32-bit data (e.g. ARGB32), it will always be aligned.
-    if (format != -1 && bitsPerPixel != 32) {
+    if (format > 0 && bitsPerPixel != 32) {
```

Comment:
According to the docs[0] 0 means "The image is invalid." so I guess both would do - an invalid image would probably not care about bitsPerPixel but setting it probably doesn't hurt or help either. 

So I guess I'll switch it to >= anyway so that it is at least consistent with all the other Qt specified formats.

[0] http://doc.qt.io/qt-5/qimage.html#Format-enum